cupy/core/core.pyx

```diff
@@ -4090,13 +4147,9 @@ cpdef inline tuple _to_cublas_vector(ndarray a, Py_ssize_t rundim):
 
 cpdef create_comparison(name, op, doc='', require_sortable_dtype=True):
 
-    if require_sortable_dtype:
```

Comment:
Would you leave this branch, because It is used in logical comparison operators?

https://github.com/cupy/cupy/blob/7ba218bfc1360672e94ef7a13d0e3993fc425989/cupy/logic/ops.py#L3

Reply:
I didn't know what to do with it, so please advise.

Reply:
Please supply `require_sortable_dtype=False` for `greater`, `greater_equal`, `less` and `less_equal` where `create_comparison` is called to make comparison ufuncs for now.

It looks that `require_sortable_dtype` is a bit confusing name and it actually works as a flag if creating a ufunc for complex numbers or not. I'll make it clear in another PR.

Reply:
Since `operator&&` and `operator||` are not defined for complex numbers yet, I think `require_sortable_dtype=False` for `logical_and` and `logical_or` in ops.py do not work.